deployment/hasura/metadata/databases/tables/sequencing/workspace.yaml

```diff
@@ -0,0 +1,50 @@
+table:
+  name: workspace
+  schema: sequencing
+configuration:
+  custom_name: "workspace"
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [name]
+      check: {}
+      set:
+        owner: "x-hasura-user-id"
```

Comment:
`updated_by: "x-hasura-user-id"` is missing from all `insert` and `update` permissions in this file.

Reply:
I don't think I fully understand this one. I see `"x-hasura-user-id"` in the `set` for inserts, and in the filter for update.

Reply:
So what is currently going on in the code highlighted for this comment is that this permission is declaring that when the action is run, regardless of what other columns are inserted, include the change "set the `owner` column to the current user". What I'm requesting is to additionally "set the `updated_by` column to the current user". This requires adding the missing `set` block to the `update` permissions. You can see an example of what I'm requesting on [L60-L87 of scheduling_goal_metadata.yaml](https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/hasura/metadata/databases/tables/scheduler/scheduling_goal_metadata.yaml#L60-L87)